docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
```

Comment:
Since scratchdata is a single process, there isn't a need to have docker-compose.yaml at the moment. I think what would be better is documenting how to run scratchdata as a docker command

Reply:
Makes sense. Do you want that `docker run` command example in this repos's README, or should I just write up a blurb you can add to the documentation (Or I could dig through the website repo to find where you have installation instructions and issue a PR there :) )